### PR TITLE
hapi-fhir-cli 4.2.0

### PR DESCRIPTION
**Pull request by shwang53**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
cp -pR /private/tmp/d20200302-86947-q49akb/hapi-fhir-4.2.0/. /private/tmp/hapi-fhir-cli-20200302-86947-121z85p/hapi-fhir-4.2.0
chmod -Rf +w /private/tmp/d20200302-86947-q49akb
Error: An exception occurred within a child process:
  Errno::EISDIR: Is a directory @ io_fread - hapi-fhir-cli
```

**Comment by mistydemeo** [comment]
Looks like 4.2.0 doesn't have an equivalent `-cli` tarball like the one we install from previously; this is trying to build from the GitHub autogenerated tarball, which doesn't have an equivalent layout.

File: Formula/hapi-fhir-cli.rb [hunk 1]
```diff
@@ -1,9 +1,8 @@
 class HapiFhirCli < Formula
   desc "Command-line interface for the HAPI FHIR library"
   homepage "https://hapifhir.io/hapi-fhir/docs/tools/hapi_fhir_cli.html"
-  url "https://github.com/jamesagnew/hapi-fhir/releases/download/v4.1.0/hapi-fhir-4.1.0-cli.zip"
```
Comment (chenrui333) [inline Formula/hapi-fhir-cli.rb]: The update on this is not correct, raised upstream issue on this

**Comment by tadgh** [comment]
in case you aren't monitoring the other issue, the tarball has been added to the releases page :)

**Comment by issyl0** [comment]
@tadgh Thanks! The build works perfectly now. :-)

**Comment by issyl0** [comment]
GitHub Actions is queued, but the Jenkins build succeeded so I'm going to merge this as is.

**Comment by chenrui333** [comment]
Thanks @tadgh, much appreciated!

**Pull request merged**